### PR TITLE
[MERL-524] Add link to migration doc in cli

### PR DESCRIPTION
**Pull request by theodesp**
## Description

Adds a link to migration path when using the `generate` script in legacy project.

## Testing
1. Run the generate script for the legacy example project:
`npm run generate -w examples/next/getting-started`

2. You should see the message in the console:
Screenshot 2022-10-10 at 18 02 48

### CODE REVIEW

**Comment by blakewilson** [comment]
@theodesp Do we want to add this to the legacy example project instead? Including this notice in the new example project may not reach out target demographic and could confuse newcomers

File: internal/faustjs.org/docs/migrationPath/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+slug: /migrationPath/overview
+title: Migrate from old version of Faust.js to Faust
```
Comment (josephfusco) [inline internal/faustjs.org/docs/migrationPath/overview.mdx:3]: Maybe specify the version of Faust that we'd like to deem the 'old version'.

File: examples/next/getting-started/gqty.config.js [hunk 2]
```diff
@@ -16,5 +16,15 @@ const config = {
 };
 
 console.log(`Using "${config.introspection.endpoint}" to generate schema...`);
+console.log(
+  '**********************************************************************',
+);
+console.log(`* ✨There is a new version of Faust ✨.                              *
+  * If you are still using the this version, and you wish to migrate,*
+  * take a look at our migration guide at                            *
+  * https://faustjs.org/docs/migrationPath/overview                  *`);
```
Comment (josephfusco) [inline examples/next/getting-started/gqty.config.js:25]:
```suggestion
console.log(`* ✨ There is a new version of Faust ✨.                             *
* If you are still using the this version, and you wish to migrate,  *
* take a look at our migration guide at                              *
* https://faustjs.org/docs/migrationPath/overview                    *`);
```

## Before
Screen Shot 2022-10-10 at 1 15 08 PM


## After
Screen Shot 2022-10-10 at 1 14 10 PM

**Review by josephfusco — COMMENTED** [review]
I like that this was added to `npm run generate`, nice work @theodesp ! Left some comments :)

**Review by josephfusco — APPROVED** [review]
LGTM!

**Pull request merged**